src/preact-portal.js

```diff
@@ -29,6 +32,8 @@ export default class Portal extends Component {
 	}
 
 	renderLayer(show=true) {
+		if (!this.isMounted) return;
```

Comment:
By adding the `setTimeout` to delay the `renderLayer` call from `componentDidUpdate` to ensure that it comes after `this.remote` is set by the `render` call in the initial `renderLayer`, it had the side effect where I was occasionally seeing the call from CDU occurring _after_ the call from `componentWillUnmount`.  That's why I put the failsafe here to ensure that no re-renders occur after this controlling component has been unmounted.

Reply:
I tried but was unable to reproduce this in a unit test.  Would love any help with that.

Reply:
Might be worth swapping this for a clearTimeout() of the timer created in CDU instead?